.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
Okay I'll merge in the latest changes and fix this up and we should be good to merge. I'll open up separate issues for things that arose in this PR.